lib/nostrum/api.ex

```diff
@@ -276,86 +290,155 @@ defmodule Nostrum.Api do
   end
 
   @doc ~S"""
-  Create a rection for a message.
+  Creates a reaction for a message.
 
-  Creates a reaction using an `emoji` for the message specified by `message_id` and
-  `channel_id`. `emoji` can be a `Nostrum.Struct.Emoji.custom_emoji.t`, a
-  base 16 unicode emoji string, or a uri encoded string.
+  This endpoint requires the `VIEW_CHANNEL` and `READ_MESSAGE_HISTORY` 
+  permissions. Additionally, if nobody else has reacted to the message with 
+  the `emoji`, this endpoint requires the `ADD_REACTIONS` permission. It 
+  fires the `MESSAGE_REACTION_ADD` event.
```

Comment:
When talking about what events are fired, do you think we should link to the event in the `Consumer` module? Their type is listed there.

Reply:
For now, I think this is a good idea. Going to put these in the next commit. I said for now, because I remembered you mentioned something about removing the Consumer module.